core/src/com/unciv/logic/civilization/managers/GreatPersonManager.kt

```diff
@@ -33,8 +35,8 @@ class GreatPersonManager : IsPartOfGameInfoSerialization {
         toReturn.freeGreatPeople = freeGreatPeople
         toReturn.greatPersonPointsCounter = greatPersonPointsCounter.clone()
         toReturn.pointsForNextGreatPersonCounter = pointsForNextGreatPersonCounter.clone()
-        toReturn.pointsForNextGreatGeneral = pointsForNextGreatGeneral
-        toReturn.greatGeneralPoints = greatGeneralPoints
```

Comment:
Please retain this for the intermediate versions, so that in multiplayer users with older version don't lose their progress
Actually in multiplayer the newer players would lose their progress, so perhaps having one version where we add the new counters and persist them...?